app/models/item.rb

```diff
@@ -6,4 +6,11 @@ class Item < ActiveRecord::Base
   validates :name, presence: true, length: { maximum: 200 }
   validates :quantity, presence: true, numericality: true
   validates :organization, presence: true
+
+  include PgSearch
+    pg_search_scope :search,
+      :against => :name,
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.